Userland/Applications/Calculator/Calculator.cpp

```diff
@@ -27,23 +27,23 @@ KeypadValue Calculator::begin_operation(Operation operation, KeypadValue argumen
         return argument;
 
     case Operation::Sqrt:
-        if (argument < KeypadValue {}) {
+        if (argument < Crypto::BigFraction {}) {
             m_has_error = true;
             return argument;
         }
-        res = argument.sqrt();
+        res = argument.inexact_sqrt();
         clear_operation();
         break;
     case Operation::Inverse:
-        if (argument == KeypadValue {}) {
+        if (argument == Crypto::BigFraction {}) {
             m_has_error = true;
             return argument;
         }
         res = argument.invert();
         clear_operation();
         break;
     case Operation::Percent:
-        res = argument * KeypadValue { 1, 2 }; // also known as `KeypadValue{0.01}`
+        res = argument * Crypto::BigFraction { 1, 2 }; // also known as `Crypto::BigFraction{0.01}`
```

Comment:
That looks incorrect. Shouldn't it be `{1, 100}`?

Reply:
Good catch!